src/components/QuestDetailsWrapper/index.tsx

```diff
@@ -68,6 +72,8 @@ export function QuestDetailsWrapper(props: QuestDetailsWrapperProps) {
 
   const questResult = useGetQuest(selectedQuestId, getQuest)
   const questMeta = questResult.data?.data
+  const projectId = questMeta?.project_id
+  const gameName = projectId && listings?.[projectId]?.project_meta?.name
```

Comment:
could we just pass in `listings?.[projectId]?.project_meta?.name` as game name as a prop to this component instead of passing the whole listings object?

Reply:
So, this was something I was talking with @eliobricenov about, I will ping your conversations about it.